examples/net1.yaml

```diff
@@ -0,0 +1,9 @@
+---
+graph: mygraph
```

Comment:
Yeah :P -- There is some good news:

The consistently named devices *are* actually useful for automation when you have more than one network card. But annoying when you just have *one*. The good news: we could easily add a function in the language that returns the eth device name if there's only one :) -- More on that in February :)

For this patch, let's just name them all `eth0` so users aren't confused about what they are. This will also work best in vms.